OperatorProblem3_latest.js

```diff
@@ -0,0 +1,62 @@
+// ***********  JavaScript Task #2 Operator Problem  **********************
+const array = [
+  "OR",
+  ["<", "a", "b"],
+  ["AND", ["==", "c", "d"], ["!=", "e", "f"]],
+];
+const newArray = [];
+const infixExpression = recursiveFunction(array); // passing given array to recursiveFunction function
+console.log("Infix Expression is:  ", infixExpression);
+// recursiveFunction function
+function recursiveFunction(element) {
+  if (Array.isArray(element)) {
+    for (let i = 0; i < element.length; i++) {
+      if (Array.isArray(element[i])) {
+        newArray.push(recursiveFunction(element[i])); // if next element is array is array then passing it to recursiveFunction function
+        if (newArray.length === 3) {
```

Comment:
In given array, pattern is that every array have 3 maximum element and these element can also be a nested arrays or  simple element. So I used this " if " check here because whenever all nested elements of an array included (array_element and simple elements) finished then they immediatley pushed into newArray. whenever newArray length equals to 3. then I poped newArray and make infix exprssion and then push it back into newArray which now only contain one infix expression element. and so on ...

Reply:
yes I removed this extra line of Code.